babel.config.js

```diff
@@ -0,0 +1,16 @@
+const presets = [
+    ["@babel/env", {
+        targets: {
+            browsers: [
+                "last 2 Chrome versions",
+                "last 2 Firefox versions",
+                "last 2 Safari versions",
+                "last 2 Edge versions",
+                "Explorer 11"
+            ]
+        },
+        useBuiltIns: "usage",
```

Comment:
This is Babel 7.x, this means babel polyfills will be included on a per-usage basis into the project build based on the target browsers above. This works a lot better than in Babel 6.x and that's why all Babel got a version bump as part of this PR.